### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by raunakkumar**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @raunakkumar @rgcostea @suneeth51 @vidhutsingh @sneharai4 @pavansshanbhag @datamattsson
+* @raunakkumar @suneeth51 @pavansshanbhag @datamattsson @dileepds @sijeesh @anushay1916
```
Comment (datamattsson) [inline CODEOWNERS:1]: It says ansushay1916 doesn't exist or doesn't have write access?

Comment (raunakkumar) [reply CODEOWNERS:1]: ya, i saw that but that was the id which was given to me @dileepds @sijeesh can you confirm?

Comment (sijeesh) [reply CODEOWNERS:1]: @datamattsson , "anushay1916" (https://github.com/anushay1916) is the username for Anusha. I can see its updated correctly in the CODEOWNERS file but you typed it incorrectly.

Comment (raunakkumar) [reply CODEOWNERS:1]: Screen Shot 2022-12-13 at 10 24 56 AM
We get the following error in the Owners file

Comment (raunakkumar) [reply CODEOWNERS:1]: Please refer to https://github.com/hpe-storage/csi-driver/pull/331

**Comment by raunakkumar** [comment]
closing in favor of https://github.com/hpe-storage/csi-driver/pull/331

**Pull request closed without merging**